CSS/Homepage.css

```diff
@@ -23,3 +30,43 @@
     align-items: center;
     height: 80px;
 }
+
+/* question-one */
+.form__list {
+    display: flex;
+    flex-direction: column;
+    min-height: 50vh;
+    justify-content: space-between;
+}
+
+.title__question {
+    margin: 0px 35px 40px 35px;
+    ;
+}
+
+.form__item {
+    margin: 0px 35px 16px 45px;
+}
+
+/*question-three*/
+.slider {
+    width: 80%;
```

Comment:
avoid giving specific width to elements, if the margins are right you should not have this problem

Reply:
I fixed the problem by placing the range inside a parent so it will have a with="100%".